content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: kind
+description: Instructions to setup kind for Istio.
+weight: 17
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare a kind cluster for Istio installation.
+
+## Prerequisites
+
+- Please use the latest Go version, ideally Go 1.13 or greater.
+- To use kind, you will also need to [install docker](https://docs.docker.com/install/).
+- Install the latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+## Installation steps
+
+1.  Create a cluster with the following command:
+
+    {{< text bash >}}
+    $ kind create cluster --name istio-testing
+    {{< /text >}}
+
+    `--name` is used to assign specific name to the cluster. By default, the cluster will be given the name `kind`.
```

Comment:
Real close now. 2 more changes that might have been missed.

to assign **a** specific name